script/script.js

```diff
@@ -42,13 +42,12 @@ dropZone.addEventListener("drop", (e) => {
 const canvasEle = document.getElementById("drawContainer");
 const context = canvasEle.getContext("2d");
 
-let startPosition = { x: 0, y: 0 };
-let lineCoordinates = { x: 0, y: 0 };
 let isDrawStart = false;
+let lines = [];
 
 const getClientOffset = (event) => {
   const { pageX, pageY } = event.touches ? event.touches[0] : event;
```

Comment:
NIT: It's just the other possible variant for this statement. To be honest, yours one looks much more clear :)
```suggestion
  const { pageX, pageY } = event?.touches?.[0] ?? event;
```